### ISSUE TITLE
Useless node in the list

### ISSUE THREAD

**Issue by matkt** [ethereum/discv4-dns-lists#6]
I was looking at the node I can find with https://github.com/ethereum/discv4-dns-lists/blob/master/all.ropsten.ethdisco.net/nodes.json. And I obviously have nodes in the list with forkids from Berlin, old chain heads of 1 month etc. I wanted to know if it's normal to have them in the list?

**Comment by fjl** [comment]
Yes it's normal. We could certainly be more strict in filtering out these nodes, but it's not implemented.

**Comment by MysticRyuujin** [comment]
We can probably close this issue now. The new crawler does fairly aggressive filtering and the lists are more optimized.
